odoo_module_migrate/migration_scripts/migrate_100_110.py

```diff
@@ -46,7 +46,7 @@
             "report.external_layout_footer is obsolete.",
     },
     ".xml": {
-        "<tree(\n|.|\t)*color=\"[^>]*":
+        ".*tree.*color":
```

Comment:
Doesn't look good. Do you have example when previous version doesn't work good?

Reply:
You can create an xml with tree node without color atribute. The regex no detect the end of file.

Reply:
Here is an example where this regexp gives false positive

https://github.com/oca/web/blob/bcc59c1785abea5ef9de4907d429784e33a62aaf/web_tree_dynamic_colored_field/views/web_tree_dynamic_colored_field.xml#L5

I think the problem with original regexp is missing ``s`` and the end and it should be ``<tree(\n|.|\t)*colors=\"[^>]*``